src/publicLinkAccess.js

```diff
@@ -38,8 +38,11 @@ class PublicFiles {
     const headers = this.helpers.buildHeaders(false)
     const url = this.getFileSharePath(tokenAndPath)
 
+    // Dont use default bearer token for public links in web
+    // this will lead to the link not being accessible
+    this.helpers.noAuth()
```

Comment:
Is this enough if it's called only here? Don't we need to call it in all the other public methods?

Reply:
@LukasHirt, From what I can tell, when you try to access a public link from a WebUI in web, it first tries to do a PROPFIND without any auth. if it gets 401 (If public link is password protected), it will give the enter password dialog. After that it uses basic auth for all other requests. So if we just handle that for the first PROPFIND request it is not really necessary for other requests.

Reply:
I see. Interesting...